lint/libs/lint-tests/src/test/java/com/android/tools/lint/checks/data/src/test/pkg/ImportFrameActivity.java.txt

```diff
@@ -10,4 +10,10 @@ public class ImportFrameActivity extends Activity {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.simple);
     }
+
+    private static class R {
+        private static class layout {
+            public static final int simple=1;
```

Comment:
Formatting: `simple = 1`.
Not very important, though, assuming it's the test data :)